lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -659,8 +658,7 @@ class HtmlViewEmbedder {
 
   /// Disposes the state of this view embedder.
   void dispose() {
-    final Set<int> allViews = PlatformViewManager.instance.debugClear();
-    disposeViews(allViews);
+    disposeViews(_viewClipChains.keys.toList());
```

Comment:
Do you need `.toList()` to avoid "modification during iteration"?

Reply:
According to tests and analyzer I only really need this in `content_manager.dart`, so I'm going to remove this one. The code should be equivalent to the call in line `412` where I removed the `.toSet`